releasenotes/notes/1.1/optimize-annotated-conjugate-reduction-656438d3642f27dc.yaml

```diff
@@ -1,4 +1,4 @@
-features:
+features_transpiler:
   - |
     Added a new reduction to the :class:`.OptimizeAnnotated` transpiler pass.
     This reduction looks for annotated operations (objects of type :class:`.AnnotatedOperation`
```

Comment:
I wonder if we want to use math mode instead of monospace mode for the letters like `B` below (I can't leave inline suggestions on that).